src/pages/Recipes/index.tsx

```diff
@@ -34,7 +34,14 @@ const Recipes = () => {
 
     // ---------------- Styles ---------------- //
     const classes = useStyles();
-    const { addRecipeBtn } = classes;
+    const { 
+        addRecipeBtn,
+        btnRecipes,
+        addRecipe,
+        ingredients,
+        ingredientsContent,
+        recipeList,
+    } = classes;
```

Comment:
if the classes are to much, don't destruct them, just use directly from the classes object like this `classes.addRecipe`

Reply:
Done